### ISSUE TITLE
Last GitHub release is outdated (so is npm/yarn database)

### ISSUE THREAD

**Issue by liarco** [kenwheeler/slick#3546]
The current version is `1.9.0`, but installing via `npm` or `yarn` results in version `1.8.1` being installed. This is probably due to the lack of the last release on GitHub.

Also the CDN note on http://kenwheeler.github.io/slick points to `1.8.1`.

Can you please update the GitHub release?

Thank you for your work.

**Comment by niftylettuce** [comment]
Please update!

**Comment by mnabialek** [comment]
Yes, I also noticed that. In sources there is 1.9.0 but it has been never released - https://github.com/kenwheeler/slick/releases

**Comment by liarco** [comment]
Latest releases were published by @leggomuhgreggo, maybe he can do something about this...

**Comment by JPustkuchen** [comment]
Confirming this problem!

**Comment by lflpowell** [comment]
Another request to publish 1.9 please.

**Comment by morgant** [comment]
_Bump._

Please at least tag 1.9.0, please.

**Comment by JPustkuchen** [comment]
Hi all, has anyone already tried to contact a maintainer? Before I'll do that I'd like to ask. Especially @liarco

**Comment by liarco** [comment]
Actually I didn't try to contact the author directly.

**Comment by morgant** [comment]
@JPustkuchen I swear I pinged @kenwheeler on Twitter around the time of my previous comment and never heard anything, of course, now can't find record of the tweet or DM. 🙁

Please do reach out to Ken as well.

**Comment by JPustkuchen** [comment]
Thank you @liarco and @morgant, I just wrote an email to Ken. Hope he won't eat me.

**Comment by sebszocinski** [comment]
Bump +1

**Comment by ahmadalfy** [comment]
Vesion 2.0.0 is now available on npm
